console.lua

```diff
@@ -280,6 +280,94 @@ commands = {{
         G:update_shop()
         return "Reset shop."
     end
+}, {
+    name = "value",
+    source = "debugplus",
+    shortDesc = "Get and modify highlighted card values",
+    desc = "Lets you get or modify the values of the currently hovered card",
```

Comment:
Since this command has sub commands it should include usage examples for the sub commands. See how tutorial and money do it.